lib/receipts/base.rb

```diff
@@ -79,11 +82,19 @@ def render_billing_details(company:, recipient:, margin_top: 16)
       table(line_items, width: bounds.width, cell_style: {borders: [], inline_format: true, overflow: :expand})
     end
 
-    def render_line_items(line_items, margin_top: 30)
+    def render_line_items(line_items:, margin_top: 30, column_widths: nil)
       move_down margin_top
 
       borders = line_items.length - 2
-      table(line_items, width: bounds.width, cell_style: {border_color: "eeeeee", inline_format: true}) do
+
+      table_options = {
+        width: bounds.width,
+        cell_style: {border_color: "eeeeee", inline_format: true}
+      }
+
+      table_options[:column_widths] = column_widths unless column_widths.nil?
```

Comment:
```suggestion
      table_options = {
        width: bounds.width,
        cell_style: {border_color: "eeeeee", inline_format: true},
        column_widths: column_widths
      }.compact
```

This would be a bit cleaner?

Reply:
i always forget about compact.